--- cssed-0.4.0-orig/src/Makefile.am	2005-09-28 04:47:07.000000000 -0500
+++ cssed-0.4.0/src/Makefile.am	2005-11-22 19:40:42.993241500 -0600
@@ -65,5 +65,5 @@
 	xpm-icons.h \
 	filetypes.h
 
-cssed_LDADD = @PACKAGE_LIBS@  ../scintilla/gtk/libscintilla.a ../libcroco/parser/libcroco.a
-cssed_LINK = g++ -DGTK -o cssed
+cssed_LDADD = ../scintilla/gtk/libscintilla.a ../libcroco/parser/libcroco.a $(PACKAGE_LIBS)
+cssed_LINK = $(CXX) $(AM_CFLAGS) $(CFLAGS) $(AM_LDFLAGS) $(LDFLAGS) -o $@
